README.md

```diff
@@ -249,6 +288,111 @@ log('Generated deeplink: ${response.deeplink}');
 log('Generated SVG QR code: ${response.svgQrCode}');
 ```
 
+#### Create Company Account
+
+*Creates a sub-account for the company.*
```

Comment:
Same thing as in the JS PR.

Replace the `sub-` first with `` then search and replace `a account` with `an account`
```suggestion
*Creates an account for the company.*
```